cloudbridge/cloud/providers/aws/resources.py

```diff
@@ -38,7 +38,6 @@
 
 
 class AWSMachineImage(BaseMachineImage):
-
```

Comment:
All of these extra lines are per PEP8 so they should remain: https://www.python.org/dev/peps/pep-0008/#blank-lines